src/steamship/app/lambda_handler.py

```diff
@@ -83,7 +91,57 @@ def _handler(event: Dict, _: Dict = None) -> Response:
             )
 
     def handler(event: Dict, context: Dict = None) -> dict:
-        response = _handler(event, context)
+        logger = None
+        loggingConfig = event.get("loggingConfig")
+
+        if loggingConfig is None:
+            return Response.error(
+                code=HTTPStatus.INTERNAL_SERVER_ERROR,
+                message="Plugin/App handler did not receive a remote logging config.",
+            ).dict(by_alias=True)
+
+        loggingHost = loggingConfig.get("loggingHost")
+        loggingPort = loggingConfig.get("loggingPort")
+
+        logger = logging.getLogger("lambda.handler")
+        logging.basicConfig(level=logging.INFO)
+        loggingHandler = None
+        # This log statement intentionally goes to the DEFAULT logging handler, to debug logging configuration issues
+        logging.info(f"Logging host: {loggingHost} Logging port: {loggingPort}")
+
+        if (
+            loggingHost != "none"
+        ):  # Key off the string none, not 'is None', to avoid config errors where remote host isn't passed
+            # Configure remote logging
+            if loggingHost is None:
+                return Response.error(
+                    code=HTTPStatus.INTERNAL_SERVER_ERROR,
+                    message="Plugin/App handler did receive a remote logging config, but it did not include a loggingHost.",
+                ).dict(by_alias=True)
+
+            if loggingPort is None:
+                return Response.error(
+                    code=HTTPStatus.INTERNAL_SERVER_ERROR,
+                    message="Plugin/App handler did receive a remote logging config, but it did not include a loggingPort.",
+                ).dict(by_alias=True)
+
+            custom_format = {
+                "level": "%(levelname)s",
+                "host": "%(hostname)s",
+                "where": "%(module)s.%(funcName)s",
+                "type": "%(levelname)s",
+                "stack_trace": "%(exc_text)s",
+            }
+            loggingHandler = fluenthandler.FluentHandler(
```

Comment:
Just thinking out loud -- is there an opportunity for us to auto-add things like:

- invoking user id
- lambda arn
- plugin/app instance id
- plugin/app version id
- plugin/app id
- space id

Etc? I realize we don't plumb some of that through to the invocation, but I kind of think it might be worth it to get really searchable logging data? 

(maybe we leave that to a future PR?)

Reply:
Absolutely, that is the intent - I wanted to get the basic end-to-end plumbing working first, and then go back and start adding bells and whistles.  I think these are some of the first things that should be added.  I have another ticket to do this work later.